typed_python/PySetInstance.cpp

```diff
@@ -236,11 +528,33 @@ void PySetInstance::copyConstructFromPythonInstanceConcrete(SetType* setType, in
                 setType->insertKey(tgt, key.data());
             }
             return;
+        } else if (PyUnicode_Check(pyRepresentation)) {
+            if (PyUnicode_READY(pyRepresentation) < 0) {
```

Comment:
why wouldn't it work to just include PyUnicodeCheck in the checks in line 495? Now that I think about it, shouldn't this work with anything that's iterable?

Reply:
@braxtonmckee , good point, rolled up PyUnicode processing to the other iterables. There was some reason why I originally spun it out, but once I rebased recently, it seems to work as part of the other iterable code path. Always feels good to clip some code and simplify.